sample-service/Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM maven:3-openjdk-11 as builder
+COPY ./sample-service/pom.xml sample-service/pom.xml
```

Comment:
Can you get the service name from the outside so that this file can be generic?

Reply:
@tamirkamara  yes, done, pls resolve.